test-case-reporting/static/test-run-list.html

```diff
@@ -1,23 +1,36 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>Test runs</title>
-</head>
-<body>
-  <ul>
-    {{#testRuns}}
-      <li>Test case is <a href="/test-results/history/{{testCase.id}}">{{testCase.name}}</a></li>
-      <ul>
-        <li>Job num is {{job.jobNumber}}</li>
-        <li>Build num is <a href="/test-results/build/{{job.build.buildNumber}}">{{job.build.buildNumber}}</a></li>
-        <li>Status is {{status}}</li>
-      </ul>
-    {{/testRuns}}
-    {{^testRuns}}
-      <h2>
-        There are no test runs.
-      </h2>
-    {{/testRuns}}
-  </ul>
-</body>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>{{title}}</title>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <!-- Surface style taken from https://github.com/niutech/amp-surface -->
+    <link rel="stylesheet" type="text/css" href="/surface.min.css"/>
+    <link rel="stylesheet" type="text/css" href="/custom.css"/>
+  </head>
+  <body>
+    <div class="g--10 m--1">
+      <div class="g--8 g-s--12 center">
+        <h1>{{title}}</h1>
+        {{#testRuns}}
+          <div class="collapsible-wrap card no-pad">
+            <input type="checkbox" id="collapsible-{{index}}">
+            <label for="collapsible-{{index}}">
+              <span class='status-badge status-{{testRun.status}}'>{{testRun.status}}</span>&nbsp;
+              <a href="/test-results/history/{{testRun.testCase.id}}">{{testRun.testCase.name}}</a>
+            </label>
+            <div class="collapsible-{{index}}-area">
+              <p>Job num is {{testRun.job.jobNumber}}</p>
```

Comment:
`Job: {{...}}`, this could link to the Travis job log
`Build: {{...}}`

Reply:
To link to the Travis job log, we need the Travis API, which would take a bit of setting up (getting the API key and so on). Seems like it would be worth it, but I'm not sure I want to start that at this moment of the internship.

We could also add links to the job & build objects and get those through the wire (they're easy to get on the Travis end), that would be simpler but less flexible than using the API if we need it for other things.

Reply:
Regardless, I would want to move that to a separate PR.

Reply:
Let's at least update the text/formatting, but leave the links as-is then

Reply:
Done.